Resources/Prototypes/Wires/layouts.yml

```diff
@@ -10,9 +10,29 @@
   - !type:DoorSafetyWireAction
 
 - type: wireLayout
-  parent: Airlock
+  parent: HighSec
   id: AirlockSecurity
```

Comment:
Think Security should keep regular airlock wires.

Reply:
fixed